make/StaticLibs.gmk

```diff
@@ -143,7 +148,53 @@ TARGETS += $(java)
 
 JAVA_LAUNCHER := $(BUILD_LAUNCHER_java_TARGET)
 
-static-launcher: $(java)
+static-launchers: $(java)
+
+################################################################################
+# Build relaunchers (thin wrappers calling the java binary) for all other
+# JDK launchers.
+################################################################################
+
+RELAUNCHER_SRC := $(TOPDIR)/src/java.base/$(OPENJDK_TARGET_OS_TYPE)/native/launcher
+
+# $1: The module name
+# $2: The launcher name
+define SetupRelauncher
+  $1_$2_LAUNCHER_ARGS_LINE :=  $$(shell cat $$(SUPPORT_OUTPUTDIR)/static-native/relaunchers/$1/$2-relauncher-arguments.txt)
```

Comment:
`$(call ReadFile, ...)`

also, double space?

Reply:
That was nicer. Had forgotten about it. Thanks!